inc/wpseo-non-ajax-functions.php

```diff
@@ -78,7 +78,8 @@ function wpseo_admin_bar_menu() {
 	}
 
 	// Yoast Icon.
-	$title = '<div class="wp-menu-image yoast-logo svg"></div>';
+	$icon_svg = WPSEO_Utils::get_icon_svg(  );
+	$title = '<div id="yoast-ab-icon" class="ab-icon wp-menu-image yoast-logo svg" style="background-image: url(\''.$icon_svg.'\') !important;"></div>';
```

Comment:
Is the `!important` necessary? Shouldn't an inline style automatically overwrite it?